### PR TITLE
feat: page blocks UI improvements

### PR DESCRIPTION
**Pull request by neatbyte-vnobis**
## Changes
Implements issue "Block selection - UI improvements"

## How Has This Been Tested?
Manual

## Documentation
None

### CODE REVIEW

**Comment by SvenAlHamad** [comment]
@neatbyte-vnobis I've tested this and overall works great, I just noticed one small thing. The two existing buttons have a tooltip on them, and the two new ones we introduced don't. Can you please add the tooltips on them so it's all unified. You can place "Refresh block" and "Edit block" as the tooltip messages. 

CleanShot 2023-03-15 at 18 55 18@2x

**Comment by neatbyte-vnobis** [comment]
@SvenAlHamad tooltips were added

File: packages/app-page-builder/src/pageEditor/config/blockEditing/SearchBlocks.tsx [hunk 1]
```diff
@@ -300,7 +351,13 @@ const SearchBar = () => {
                             <SimpleFormHeader
                                 title={categoryPlugin.title}
                                 icon={<IconWrapper>{categoryPlugin.icon}</IconWrapper>}
-                            />
+                            >
+                                <IconButton
+                                    icon={<RefreshIcon />}
+                                    onClick={refetchBlocks}
+                                    data-testid={"pb-page-details-header-edit-revision"}
```
Comment (adrians5j) [inline packages/app-page-builder/src/pageEditor/config/blockEditing/SearchBlocks.tsx]: Incorrect `testid` name. But feel free to just remove it for now.

**Review by adrians5j — CHANGES_REQUESTED** [review]
One super minor thingie, other than that, we can merge.

**Comment by neatbyte-vnobis** [comment]
@adrians5j last comment is fixed now

**Pull request merged**